[dynamo] prevent recompilations on external_utils.wrap_inline

williamwen42 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack/tree/0.15.0) (oldest at bottom):
* #171826
* __->__ #163503
* #185524
* #186100
* #191389
* #191804

williamwen42 committed:
[dynamo] prevent recompilations on external_utils.wrap_inline

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilations on external_utils.wrap_inline"


This will prevent recompilations due to wrap_inline's guard on `fn` from hitting the recompilation limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

StrongerXi reviewed (approved):
I'd like to understand why we are hitting this problem for `wrap_inline`, etc. now. Because in theory this is a dynamo problem for all non-inplace function wrappers.

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 commented:
@pytorchbot rebase

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] prevent recompilation limit exceeded on external_utils.wrap_inline"


This will prevent recompilations (i.e. logs to TORCH_LOGS="recompiles") due to wrap_inline's guard on `fn`. If we call wrap_inline on enough different functions, we would hit the recompile limit.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx kadeng chauhang amjames Lucaskabela jataylo chenyang78

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update

[ghstack-poisoned]
